src/worker/helpers/shmem_helper.py

```diff
@@ -0,0 +1,62 @@
+from multiprocessing import shared_memory
+from multiprocessing import resource_tracker
+import sys
+
+"""Moneo Shared Memory Helper"""
+
+
+def clean_Leaked_shm(shm_name):
+    ''' Clean up Resources'''
+    shm = shared_memory.SharedMemory(name=shm_name)
+    shm.close()
+    shm.unlink()
+
+
+class Shared_Mem_Mngr:
+    """Moneo Shared Memory Helper class"""
+
+    def __init__(self, shm_name, isClient=False):
+        '''Init, set shm name and type of Access type Client/Server.'''
+        self.isClient = isClient
+        self.name = shm_name
+        self.shmCreated = False
+
+    def create_shm(self, data=None, size=1024):
+        '''Create memory of data size or of specified size. If data assign data.'''
+        if self.isClient:  # client does not need to create shared mem just retrieve
+            shm = self.get_shm()
+        else:
+            if self.shmCreated:
+                self.delete_shm()  # delete old shared memory
+            if data:  # If data provided creat shm of size data and assign
+                size = len(data.encode("utf8"))
+                shm = shared_memory.SharedMemory(
+                    create=True, size=size, name=self.name)
+                shm.buf[:size] = bytes(data, 'UTF-8')
+            else:  # Create shm of specified size
```

Comment:
Should there be a default value written to the shared memory block upon creation? It would match up better with what happens when data is provided that way. Up to you.

Reply:
I rather leave it like this and let the caller control this. The module is supposed to be generalized so who knows what is supposed to be in the memory. In reality it will be job data. I rather keep it generalized in case it has another use